vertical-pod-autoscaler/pkg/admission-controller/logic/server.go

```diff
@@ -147,7 +147,7 @@ var (
 	}
 )
 
-func validateVPA(vpa *vpa_types.VerticalPodAutoscaler) error {
+func validateVPA(vpa *vpa_types.VerticalPodAutoscaler, isCreate bool) error {
```

Comment:
nit: I am inclined to pass ar.Request.Operation instead of boolean but you can ignore my inclination :)

Reply:
I'll choose "ignore" this time.